src/lib/types.ts

```diff
@@ -262,7 +262,7 @@ export interface VerifyMobileOTPParams {
   email?: undefined
   phone: string
   token: string
-  type?: MobileOTPType
+  type: MobileOTPType
```

Comment:
this was optional in the past for backward compatibility reasons because we didn't support email otps so the user didn't have to pass a type since it would always be `type = sms`